lib/pages/settings/view/ask_permissions.dart

```diff
@@ -0,0 +1,114 @@
+import 'package:acs_upb_mobile/authentication/model/user.dart';
+import 'package:acs_upb_mobile/authentication/service/auth_provider.dart';
+import 'package:acs_upb_mobile/generated/l10n.dart';
+import 'package:acs_upb_mobile/pages/settings/model/request.dart';
+import 'package:acs_upb_mobile/pages/settings/service/request_provider.dart';
+import 'package:acs_upb_mobile/resources/utils.dart';
+import 'package:acs_upb_mobile/widgets/scaffold.dart';
+import 'package:acs_upb_mobile/widgets/toast.dart';
+import 'package:flutter/gestures.dart';
+import 'package:provider/provider.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+
+class AskPermissions extends StatefulWidget {
+  static const String routeName = '/askPermissions';
+
+  @override
+  State<StatefulWidget> createState() => AskPermissionsState();
+}
+
+class AskPermissionsState extends State<AskPermissions> {
+  User user;
+  String requestBody = "";
+  bool agreedToResponsabilities = false;
+
+  _fetchUser() async {
+    AuthProvider authProvider = Provider.of(context, listen: false);
+    user = await authProvider.currentUser;
+    if (mounted) {
+      setState(() {});
+    }
+  }
+
+  initState() {
+    super.initState();
+    _fetchUser();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    RequestProvider requestProvider =
+        Provider.of<RequestProvider>(context, listen: false);
+
+    return AppScaffold(
+        title: S.of(context).navigationAskPermissions,
+        actions: [
+          AppScaffoldAction(
+              icon: Icons.save,
```

Comment:
It is generally better to not use icons without text, especially for actions (https://ux.stackexchange.com/questions/1795/when-to-use-icons-vs-icons-with-text-vs-just-text-links). Use a "send" text label instead.